doc/ref_model/chapters/chapter01.md

```diff
@@ -122,11 +122,12 @@ This document focuses on the documenting the higher level concepts that are need
 This document specifies:
 - **NFVI Infrastructure abstraction**: in context with how it interacts with the other components required to build a complete system that supports **VNF**s.
   - **NFVI metrics & capabilities**: A set of metrics and capabilities for the NFVI which VNFs require to perform telco scale network functions.
-  - **Infrastructure profiles catalogue**: A catalogue of standard profiles needed in order to completely abstract the infrastructure from VNFs. With a limited and well-defined set of profiles with well understood characteristics, VNF compatibility and performance predictability can be achieved.
+  - **Infrastructure profiles catalogue**: A catalogue of standard NFVI Profiles needed in order to completely abstract the infrastructure from VNFs. With a limited and well-defined set of profiles with well understood characteristics, VNF compatibility and performance predictability can be achieved.
```

Comment:
To me, an NFVI profile defines how the compute host and the underlay leaf switch needs to be configured in support of a flavor.  The VNFs shouldn't be aware of the NFVI profile (both software and hardware). If we are using the NFVI profile to mean something else then there are impacts on RA-1 chapters.

Reply:
Was there a change suggested?

Reply:
I think the text is OK.